### PR TITLE
#821 Audit and make documentation for command GETEX consistent

### PR DESCRIPTION
**Pull request by AbhirajPatwa**
Audited GETEX documentation to be consistent to redis #821

### CODE REVIEW

**Comment by AbhirajPatwa** [comment]
For issue #821

File: docs/src/content/docs/commands/GETEX.md [hunk 1]
```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
...
 
 ### Example 3: Retrieve value and set expiration as Unix timestamp in seconds
 
-```DiceDB
-SET mykey "Hello"
-GETEX mykey EXAT 1672531199
+```bash
+127.0.0.1:7379> SET mykey "Hello"
+OK
+127.0.0.1:7379> GETEX mykey EXAT 1672531199
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETEX.md:82]:
```suggestion
127.0.0.1:7379> GETEX mykey EXAT 1672531199
127.0.0.1:7379> "Hello"
```

File: docs/src/content/docs/commands/GETEX.md [hunk 2]
```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
...
 
 ### Example 4: Retrieve value and set expiration as Unix timestamp in milliseconds
 
-```DiceDB
-SET mykey "Hello"
-GETEX mykey PXAT 1672531199000
+```bash
+127.0.0.1:7379> SET mykey "Hello"
+OK
+127.0.0.1:7379> GETEX mykey PXAT 1672531199000
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETEX.md:92]:
```suggestion
127.0.0.1:7379> GETEX mykey PXAT 1672531199000
127.0.0.1:7379> "Hello"
```

File: docs/src/content/docs/commands/GETEX.md [hunk 3]
```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
...
 
 ### Example 2: Retrieve value and set expiration in milliseconds
 
-```DiceDB
-SET mykey "Hello"
-GETEX mykey PX 10000
+```bash
+127.0.0.1:7379> SET mykey "Hello"
+OK
+127.0.0.1:7379> GETEX mykey PX 10000
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETEX.md:72]:
```suggestion
127.0.0.1:7379> GETEX mykey PX 10000
127.0.0.1:7379> "Hello"
```

File: docs/src/content/docs/commands/GETEX.md [hunk 4]
```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
...
 ### Example 1: Retrieve value and set expiration in seconds
 
-```DiceDB
-SET mykey "Hello"
-GETEX mykey EX 10
+
+```bash
+127.0.0.1:7379> SET mykey "Hello"
+OK
+127.0.0.1:7379> GETEX mykey EX 10
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETEX.md:62]:
```suggestion
127.0.0.1:7379> GETEX mykey EX 10
127.0.0.1:7379> "Hello"
```

File: docs/src/content/docs/commands/GETEX.md [hunk 5]
```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
...
-```DiceDB
-SET mykey "Hello"
-EXPIRE mykey 10
-GETEX mykey PERSIST
+```bash
+127.0.0.1:7379> SET mykey "Hello"
+OK
+127.0.0.1:7379> EXPIRE mykey 10
+OK
+127.0.0.1:7379> GETEX mykey PERSIST
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETEX.md:104]:
```suggestion
127.0.0.1:7379> GETEX mykey PERSIST
127.0.0.1:7379> "Hello"
```

File: docs/src/content/docs/commands/GETEX.md [hunk 6]
```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
 3. `Invalid expiration time`: If the expiration time is not a valid integer or timestamp, it will return an error.
    - Error message: `(error) ERR value is not an integer or out of range`
 
-## Example Usage
+## Examples
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETEX.md:54]: Let's keep it consistent with what we have in other docs. In future, we can rename all of them in one PR to make them consistent. 
```suggestion
## Example Usage
```

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Hi, Thanks for making all of these changes. I have left few comments.

**Comment by AbhirajPatwa** [comment]
Added your part, should we merge now?

**Review by apoorvyadav1111 — APPROVED** [review]
Thanks for the changes, Approved.

**Pull request merged**